docs/pages/changelog/index.js

```diff
@@ -37,19 +59,17 @@ export class Changelog extends LitElement {
   render() {
     return wrap({
       title: "Changelog",
-      intro:
-        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ac turpis accumsan, maximus purus sit amet, suscipit elit. Vivamus egestas a diam a luctus. Sed et viverra nibh. Nam sed ipsum felis.",
       children: html`
         <section is="vl-region">
           <div is="vl-layout">
             <div is="vl-grid" data-vl-is-stacked>
-              ${change({
+              <!-- ${change({
```

Comment:
Comments in final release?

Reply:
Voorlopig in comments omdat er niets unreleased is, maar ik het ook niet wou weggooien omdat er binnenkort wel dingen unreleased zijn. We zouden daar eventueel iets op kunnen bedenken dat we ergens 'unreleased' true of false kunnen zetten.

Reply:
Ok :)